web/styles/popovers.css

```diff
@@ -236,6 +237,10 @@
         margin: 0 !important;
     }
 
+    .popover-stream-info-menu-description {
+        color: var(--color-text-popover-menu) !important;
```

Comment:
Which selector is this fighting with that `!important` is necessary here? Since you're adding new classes in this area, I'd like to see if we can figure out the specificity issue and tackle it directly.

Reply:
Yes, `!important` would be necessary here if we want `var(--color-text-popover-menu)` color for our description.
The selector it is fighting against is `.popover-menu .text-item`.
<details><summary>Without `!important` the following property overrides the description color.</summary>
<p>

![image](https://github.com/user-attachments/assets/2e288bbb-f50a-4cdd-91c4-4880e442dad7)


</p>
</details> 

-------------------------------------------------------------------------------------------------------------------------------

| With `!important` | Without `!important` |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/096c25be-c779-4328-95f7-d1e451fec67e) | ![image](https://github.com/user-attachments/assets/bf22ecf1-34b3-4d60-a539-cf7195ecb023) | 

Do let me know which one seems good to you, will update my PR accordingly.

Reply:
Would that be resolved by referencing `#stream-info-popover`, as an ID, instead of as the class `.stream-info-popover`? Looks like the ID pattern is already established with `#stream-actions-menu-popover`.

Reply:
Oh yes, you are correct. By referencing it as an ID instead of a class, we no longer need `!important`!
I will update my PR. Thanks @karlstolley!